engine/dlib/src/dmsdk/dlib/dns.h

```diff
@@ -55,10 +55,11 @@ namespace dmDNS
     /*# create a new channel
      * Creates a new channel that can be used for DNS queries.
      * @param channel [type:dmDNS::HChannel*] Pointer to the created channel if successful, will be left alone otherwise
+     * @param servers [type:char*] Comma separated list of DNS servers to use, null or empty string to use network default.
      * @name dmDNS::NewChannel
      * @return RESULT_OK on succcess
      */
-    Result NewChannel(HChannel* channel);
+    Result NewChannel(HChannel* channel, const char* servers);
```

Comment:
This is a breaking change. (dmSDK header!)

Generally we try to avoid those, but since it's a new api it might be fine, since we only have to update one extension, and I don't think there are any other extensions using this api.

For future reference, other options to consider in cases like this:
* Set-function
* Different constructor

Reply:
Good point!

What do we prefer?

    void SetServers(const char* servers)

Or

    Result NewChannelWithServers(HChannel* channel, const char* servers);

Reply:
I went for `SetChannelServers`